### PR TITLE
[WFLY-15544] iiop-openjdk subsystem transformation to Jakarta

### PR DESCRIPTION
**Pull request by tadamski**
https://issues.redhat.com/browse/WFLY-15544

### CODE REVIEW

**Comment by tadamski** [comment]
retest this please

**Comment by darranl** [comment]
The build error seems to be:

[ERROR] Failed to execute goal on project wildfly-iiop-openjdk-jakarta: Could not resolve dependencies for project org.wildfly:wildfly-iiop-openjdk-jakarta:jar:26.0.0.Final-SNAPSHOT: Could not find artifact org.jboss.narayana.jts:narayana-jts-idlj-jakarta:jar:5.12.1.Final in brontes-mirror (http://repository.ci.wildfly.org/content/groups/public/) -> [Help 1]
01:03:04   [ERROR]

I suspect we need the poms updating following the preparation of the recent release.

**Comment by darranl** [comment]
Just trying CI once more.

**Comment by tadamski** [comment]
It seems that the tests are passing but there are too many ignored ones and hence the failures in the build.

**Comment by bstansberry** [comment]
This is failing CI because narayana-jts-idlj-jakarta doesn't exist in the Narayana 5.12.1 release.

https://search.maven.org/artifact/org.jboss.narayana.jts/narayana-jts-idlj-jakarta

#14974 is to move to 5.12.4 but @mmusgrov noted there's a perf problem with that release. I'll ask there if 5.12.2 is a suitable interim step to unblock this one.

**Comment by bstansberry** [comment]
@tadamski The needed Narayana upgrade is in, but this needs conflicts resolved.

**Comment by tadamski** [comment]
@bstansberry rebased and fixed

File: ee-9/source-transform/iiop-openjdk/pom.xml [hunk 1]
```diff
@@ -0,0 +1,150 @@
...
+        </dependency>
+
+        <dependency>
+            <groupId>org.wildfly.core</groupId>
+            <artifactId>wildfly-network</artifactId>
+        </dependency>
+
+        <dependency>
+            <groupId>org.jboss.metadata</groupId>
+            <artifactId>jboss-metadata-common</artifactId>
```
Comment (yersan) [inline ee-9/source-transform/iiop-openjdk/pom.xml:86]: You should use `jboss-metadata-common-jakarta` instead and move it up to the Jakarta-namespace specific deps section of your pom.

File: ee-9/source-transform/iiop-openjdk/pom.xml [hunk 2]
```diff
@@ -0,0 +1,150 @@
...
+        </dependency>
+
+        <dependency>
+            <groupId>org.jboss.metadata</groupId>
+            <artifactId>jboss-metadata-ejb</artifactId>
```
Comment (yersan) [inline ee-9/source-transform/iiop-openjdk/pom.xml:91]: You should use `jboss-metadata-ejb-jakarta` instead and move it up to the Jakarta-namespace specific deps section of your pom.

**Review by yersan — CHANGES_REQUESTED** [review]

File: ee-9/source-transform/iiop-openjdk/pom.xml (hunk 1 above)
Comment (tadamski) [reply ee-9/source-transform/iiop-openjdk/pom.xml]: fixed

File: ee-9/source-transform/iiop-openjdk/pom.xml (hunk 2 above)
Comment (tadamski) [reply ee-9/source-transform/iiop-openjdk/pom.xml]: fixed

**Pull request merged**